src/actions/websocket.test.js

```diff
@@ -430,3 +439,106 @@ describe('Actions.Websocket', () => {
         test();
     });
 });
+
+describe('Actions.Websocket doReconnect', () => {
+    const mockStore = configureMockStore([thunk]);
+
+    const currentTeamId = 'team-id';
+    const currentUserId = 'user-id';
+    const currentChannelId = 'channel-id';
+
+    nock(Client4.getBaseRoute()).
```

Comment:
I believe you also need to mock the other endpoints or it will fail to run the tests